packages/cli/test/sim/backupEthProvider.test.ts

```diff
@@ -1,11 +1,11 @@
 /* eslint-disable @typescript-eslint/naming-convention */
 import path from "node:path";
 import {activePreset} from "@lodestar/params";
-import {SimulationEnvironment} from "../utils/simulation/simulationEnvironment.js";
-import {nodeAssertion} from "../utils/simulation/assertions/nodeAssertion.js";
```

Comment:
Assertion is one step of the whole process that involves in it, capturing data, asserting logic, dumping raw data for debugging. 

So why I tend to use a broader term here.

Reply:
Scenario to me is like "range syncing" or similar.... doesn't speak to "collecting information and asserting against it"

Reply:
as in "a scenario" is the "range syncing scenario".... and within a scenario there are a number of data points that are collected and asserted against.  the collection and asserting is "an assertion" to me, not "a scenario".  like there are assertions that happen at the slot, there are assertions that happen at the epoch.  not there are scenarios that happen at the slot and scenarios that happen at the epoch... i think "scenario" is the wrong work for this construct and prefer it the way it was before the change.  I think the wording "Assertion" was more descriptive before.

Reply:
Ok let's get some more opinion on it as well.

Reply:
By looking at the existing implementations we have e.g. `merge` or `accountBalance` or `withdrawals`. 

I still feel more comfortable with the `mergeScenario` vs `mergeAssertion`, or `withdrawalsScenario` vs `withdrawalAssertion` as the later one not cover the whole scope of what is implemented in it. 

It's just the linguistic semantics, not would be better to use what is clear to more people.

Reply:
Agreed.  I think the whole flow of collect data, check assertion and store result is an "assertion"..... for merge it would be a `MergeAssertion` for blobs it would be a "BlobAssertion".  Like a blob assertion in my mind is the hook to send the blob transaction, to wait scheduling for the blob to be propagated, the calling of the API to get the blobs from the CL and then the assert statement to check the blob that was received is the same as the blob that was sent... That whole workflow is an "assertion".  I suppose "Test" would also be appropriate because the test can contain several "assertions".  But a "scenario" is more something the talks about the flow or conditions of the simulation as a whole.

Reply:
I agree with @matthewkeil. Open to other ideas, but between those two, assertion is more helpful here.

Reply:
> mergeScenario vs mergeAssertion

Looking at this in the code, this is definitely an asssertion as it asserts that the merge transition was successful. A "merge scenario" sounds like a configuration you would pass to the simulation framework to the merge as a whole.

Reply:
So be it assertion then.